src/shell/lisa_shell

```diff
@@ -373,6 +373,69 @@ echo
 
 
 ################################################################################
+# LISA Workloads utility functions
+################################################################################
+
+function lisa-wltest-init {
+export LISA_VENV="$LISA_HOME/tools/wltests/wa3.venv"
+export LISA_WAUTO="$LISA_HOME/tools/wltests/wlauto"
```

Comment:
Looks like you mean WLAUTO. Also, "wlauto" seems to be an unused abbreviation - I would say use LISA_WA_REPO or something, and call it "workload-automation" or "wa.

Reply:
Let's go for:
```
export LISA_WA="$LISA_HOME/tools/wltests/wa"
```